docs/_config.yaml

```diff
@@ -13,5 +13,5 @@ header_pages:
   - setup_instructions/setup_overview.md
   - workshop-schedule.md
 
-
-
+# Workshop type: rrp or openrrp
+workshop: "rrp"
```

Comment:
Quotes shouldn't be needed.
```suggestion
workshop: rrp
```

Reply:
I wonder if we want to be more modular and maybe use something like this instead?
```suggestion
include-openrrp: false
```

Reply:
I like this!